distance_tests.ipynb

```diff
@@ -1,28 +1,28 @@
 {
```

Comment:
<p>Can we add an interpretation of the plot below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>